back/engines/commercial/user_custom_fields/spec/models/user_custom_fields/representativeness/ref_distribution_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe UserCustomFields::Representativeness::RefDistribution do
+  subject(:ref_distribution) { build(:ref_distribution) }
+
+  describe 'factory' do
```

Comment:
I have seen tests like this in other parts of the code base. I think it is strange. This test does not describe anything about the described class. It verifies that the factory produces a valid model, which should not be tested here imo.

Reply:
I understand your point, but consistency first I guess. That's something you might want to bring up during the CDM.

Reply:
At some point, I will :-)